lib/contentful_middleman/tools/backup.rb

```diff
@@ -0,0 +1,84 @@
+require 'fileutils'
+
+module ContentfulMiddleman
+  module Tools
+    class NullBackup
+      def restore; end
+      def destroy; end
+    end
+
+    class Backup
+
+      class << self
+        def basepath
+          ::File.join ENV["MM_ROOT"], ".tmp", "backups"
+        end
+
+        def ensure_backup_path!
+          return if ::File.exists? basepath
+
+          FileUtils.mkdir_p basepath
+        end
+      end
+
+      ensure_backup_path!
```

Comment:
Don't why I did that. Changed here https://github.com/madtrick/contentful_middleman/commit/67f874d328f7c3b8e9daba91396359074c48110b